jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/testSuite/JwalaTest.java

```diff
@@ -0,0 +1,62 @@
+package com.cerner.jwala.ui.selenium.testSuite;
+
+import com.cerner.jwala.ui.selenium.JwalaResourceChromeTestSuite;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Wraps Jwala specific UI actions e.g. click tab, etc also contains methods used commonly in tests
+ * Created by Jedd Cuison on 2/28/2017
+ */
+public class JwalaTest {
```

Comment:
Since we're hard coding the Chrome Web Driver in this class we should modify the name of the class to indicate that it currently only works with Chrome